integration/src/test/scala/com/spotify/scio/bigquery/TypedBigQueryIT.scala

```diff
@@ -186,6 +188,22 @@ class TypedBigQueryIT extends PipelineSpec with BeforeAndAfterAll {
     }
   }
 
+  it should "write case classes using Storage Write API" in {
+    runWithRealContext(options) { sc =>
+      sc.parallelize(records)
+        .saveAsTypedBigQueryTable(
+          typedTableStorage,
+          createDisposition = CREATE_IF_NEEDED,
+          method = WriteMethod.STORAGE_WRITE_API
+        )
+    }.waitUntilFinish()
+
+    runWithRealContext(options) { sc =>
+      val data = sc.typedBigQuery[Record](typedTableStorage)
+      data should containInAnyOrder(records)
+    }
+  }
+
```

Comment:
Both of these tests are failing. `LocalTime` fields are being written incorrectly using the Storage Write API. For example, for the same record, FileLoads will write the value `19:00:00` and StorageWrite will write the value `15:59:15.338944`. Not quite sure what's going on, needs investigation.